addons/sale/models/sale.py

```diff
@@ -402,6 +403,28 @@ def _onchange_commitment_date(self):
                 }
             }
 
+    @api.onchange('pricelist_id', 'order_line')
+    def _onchange_pricelist_id(self):
+        if self.order_line and self._origin.pricelist_id and self._origin.pricelist_id != self.pricelist_id:
```

Comment:
`if self.order_line and self.pricelist_id and ...
`
The button shouldn't show up when the pricelist field is still empty.
![image](https://user-images.githubusercontent.com/19364378/69745188-09e90000-1142-11ea-9355-b48777cc7be9.png)

Reply:
@Feyensv Done.